build/media-suite_compile.sh

```diff
@@ -1980,6 +1980,7 @@ if [[ $mpv != "n" ]] && pc_exists libavcodec libavformat libswscale libavfilter;
             done
         fi
 
+        extra_script pre configure
```

Comment:
Haven't thought of this until now, but do the functions have access to the variables in this scope? i.e. can they add/remove stuff to `mpv_cflags` or `mpv_ldflags`?

Reply:
Yea, it seems they do

Reply:
they should have access to all of the variables within that sections

Reply:
Cool.